### ISSUE TITLE
No longer works

### ISSUE THREAD

**Issue by siddrajput** [hjdhjd/homebridge-myq#216]
**Describe The Problem:**
Garage door won't open. It used to work but recently (since about 2 weeks) it stopped working.

**To Reproduce:**
Open the garage door from the iPhone Home app.

**Logs:**

```
[12/28/2020, 06:09:38] [myQ] myQ API: Connection has been reset.
[12/28/2020, 06:09:38] [myQ] myQ API: Unable to authenticate. Will retry later.
```

**Homebridge Configuration:**

```json
        {
            "email": "---",
            "password": "---",
            "platform": "myQ"
        },
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** 1.2.5
* **Node Version:** v12.18.2
* **Homebridge-myQ Plugin Version**: homebridge-myq2 v2.1.12
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iPhone XR iOS14.2
* **Operating System and OS Version:** Raspbian GNU/Linux Stretch (9)

**Comment by donavanbecker** [comment]
@siddrajput, Your myQ version is 2.1.12 and should be v2.4.0

It also looks like your still running the homebridge-myq2 plugin which was changed into homebridge-myq, I believe right after v2.1.12.

So you will want to install the latest version of the plugin. and install the old version.

**Comment by hjdhjd** [comment]
You’re running a significantly outdated version of the plugin @siddrajput - as @donavanbecker suggested, I’d uninstall `homebridge-myq2` and install `homebridge-myq` and all should be right with the world.

**Comment by kbrashears5** [comment]
I was having the same issue - my garage door showed open in homekit when it was actually closed. myq app confirmed that it thinks the door is closed as well. Found the aforementioned logs from @siddrajput in my instance as well.

I upgraded to latest version @2.4.0 and restarted my homebridge and appears to be working again, we will see if it happens again.

I will say the version I was using was pretty recent - I think maybe 2.3.5? (I check for upgrades on all homebridge packages maybe every other week)

Guessing it was probably a hiccup in myQ servers to where auth was unable to be reacquired.
